Src/Delivery/studio/built_ins/classic/ISE_RUNTIME.e

```diff
@@ -0,0 +1,81 @@
+class
+	ISE_RUNTIME
+
+feature -- Feature specific to ISE runtime.
+
+	generator_of_type (a_type_id: INTEGER): STRING
+			-- Name of the generating class of current object
+		external
+			"C inline use %"eif_out.h%""
+		alias
+			"return c_generator_of_type(eif_decoded_type($a_type_id));"
```

Comment:
Is there any need for the instance_free postcondition ?
or this is really part of the original class?

Reply:
Good question, I've checked other built-ins and I didn't see they use the instance free postcondition,  the original class has the instance free postcondition.
Similarly, some features are marked as frozen in the original class and I didn`t use it in the built-in class.

Reply:
I would say ... if the compiler accepts the frozen and the assertion in the built-in classes, then add them. At worst they will be ignored.
But be sure to keep them in the original Eiffel class.